ui/packages/catalog-ui-search/src/main/webapp/react-component/menu/menu.tsx

```diff
@@ -112,10 +155,10 @@ export class Menu extends React.Component<any, any> {
     )
 
     return (
-      <Component>
+      <MenuRoot className={this.props.className}>
```

Comment:
What do you need className for?

Reply:
Other components using this are still relying on less - this allows us to use styled without breaking others.